scripts/ci/deploy.sh

```diff
@@ -19,18 +19,37 @@
 set -e
```

Comment:
Ah yes, I forgot that this deploys both production and snapshot builds. In that case perhaps `deploy-snapshot.sh` would be a better name than `deploy-aws-s3.sh` ? \*shrug\*